conan/tools/microsoft/toolchain.py

```diff
@@ -52,27 +61,23 @@ def __init__(self, conanfile):
         self.cflags = []
         self.ldflags = []
         self.configuration = conanfile.settings.build_type
+        self.platform = arch_to_vcproj_platform(str(conanfile.settings.arch))
```

Comment:
I've exposed platform attribute in MSBuildToolchain because it was exposed in MSBuildDeps and they share the same underlying logic, but I'm not sure it makes sense. Is it really customizable in MSBuild files?

Reply:
I am ok with it, I think it is customizable, but in the worst case, it is not critical that it is exposed, just don't change it if you don't need to change it.